packages/terra-clinical-detail-view/src/DetailView.jsx

```diff
@@ -118,19 +121,19 @@ const DetailView = (props) => {
   }
 
   return (
-    <div {...attributes}>
+    <section {...attributes}>
       <div className={cx('titles-section', { 'titles-smaller': isSmallerTitles })}>
         {titleElement}
```

Comment:
This section element should have an aria-labelledby pointing to an id value that is on the heading.  section elements aren't announced unless they have an accessible name associated.  Using aria-labelledby in this way creates the accessible name for the section content.

Reply:
The `title` prop is optional. We may not be able to tie this to the id in the title if it's not passed in. We also need to include id in this PR https://github.com/cerner/terra-clinical/pull/858/files#diff-890b5219922c74bbe07ef699395ae882c595463b51f77a947d6c0fe5ad8e7e0bR36

Reply:
As noted previously, we'll add the title prop to a new Jira.